srtcore/api.cpp

```diff
@@ -2835,17 +2835,18 @@ int CUDT::configureGroup(SRTSOCKET groupid, const char* str)
 
 int CUDT::getGroupData(SRTSOCKET groupid, SRT_SOCKGROUPDATA* pdata, size_t* psize)
 {
-    if ( (groupid & SRTGROUP_MASK) == 0)
+    if ((groupid & SRTGROUP_MASK) == 0 || !psize)
     {
         return APIError(MJ_NOTSUP, MN_INVAL, 0);
     }
 
```

Comment:
I think if the psize is already found out as provided, it should be at least set to 0, provided that it should be now granted to be set to the proper size, if provided. This is required for a user to distinguish the usage error from inability to fill the array.

Reply:
🤔